okhttp-kotlin/src/main/kotlin/okhttp3/kotlin/kotlin.kt

```diff
@@ -0,0 +1,23 @@
+package okhttp3.kotlin
+
+import okhttp3.HttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+
+fun request(url: String? = null, init: Request.Builder.() -> Unit = {}) = Request.Builder().apply {
```

Comment:
Is a top level fun desirable for the lib?

Reply:
It is top level within okhttp3.kotlin package.  But what would you suggest instead?